bin/ch-test

```diff
@@ -1180,6 +1202,16 @@ case $phase in
             dir_perm_rm "$d"
         done
         ;;
+    rootemu)
+        if [[ $scope != full ]]; then
+            if [[ -z "$scope" ]]; then
+                scope="standard"
+            fi
+            printf "error: rootemu incompatible with scope: %s\n" "$scope"
+            exit 1
+        fi
```

Comment:
I think we want `rootemu` to be *possible* in standard scope, just not the default.